sites/friday/src/manage-resources/resource-init.md

```diff
@@ -106,79 +172,223 @@ platform integration:add \
-| Kafka                   | 0.1  | 448 MB | 512 MB  |
-| Lisp                    | 0.1  | 64 MB  | 0 MB    |
-| MariaDB                 | 0.1  | 448 MB | 256 MB  |
-| Memcached               | 0.1  | 352 MB | None    |
-| MongoDB                 | 0.1  | 448 MB | 256 MB  | 
-| MongoDB Premium         | 0.1  | 448 MB | 256 MB  |
-| Network Storage         | 0.1  | 448 MB | 256 MB  | 
-| NodeJS                  | 0.1  | 64 MB  | 0 MB    |
-| OpenSearch              | 0.1  | 448 MB | 256 MB  | 
-| Oracle Java             | 0.1  | 448 MB | 0 MB    |
-| Oracle MySQL            | 0.1  | 448 MB | 256 MB  | 
-| PHP                     | 0.1  | 64 MB  | 0 MB    |
-| PostgreSQL              | 0.1  | 448 MB | 256 MB  | 
-| Python                  | 0.1  | 64 MB  | 0 MB    |
-| Rabbitmq                | 0.1  | 448 MB | 256 MB  | 
-| Redis ephemeral         | 0.1  | 352 MB | None    |
-| Redis persistent        | 0.1  | 352 MB | 256 MB  | 
-| Ruby                    | 0.1  | 64 MB  | 0 MB    |
-| Rust                    | 0.1  | 64 MB  | 0 MB    |
-| Solr                    | 0.1  | 448 MB | 256 MB  | 
-| Varnish                 | 0.1  | 448 MB | None    |
-| Vault KMS               | 0.1  | 448 MB | 256 MB  |
+{{% note theme="info" title="Prerequisite" %}}
 
-\* The disk size is set to `None` when the container never uses a disk, and to `0 MB` when the container doesn't require a disk but can use one.
+Make sure `sizing_api_enabled` is set to `true` in your `{{% vendor/configfile "app" %}}` file.
 
-## Actions on environments
+{{% /note %}}
 
-Here is how resource allocation works on {{% vendor/name %}} when you perform the following actions on an environment:
+{{< codetabs >}}
 
-### Environment creation
++++
+title=Using the CLI
++++
 
-When you [branch an environment](/glossary.md#branch) to create a new child environment,
-the child environment inherits all the resources from its parent.
+Run the following command:
+
+```bash {location="Terminal"}
+environment:branch --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
+```
+
+For example, to use the `minimum` resource initialization strategy, run the following command:
+
+```bash {location="Terminal"}
+environment:branch --resources-init=minimum
+```
+
+<--->
+
++++
+title=In the Console
++++
+
+1. Navigate to the environment you want to branch from.
+2. Click {{< icon branch >}} **Branch**.
+3. Enter a name and a type for the new environment.
+4. Select an initialization strategy from the proposed list.
+5. Click **Branch**.
+
+{{< /codetabs >}}
 
 ### Environment merge
 
 When you [merge](/glossary.md#merge) a child environment into a parent environment,
 any app or service on the child environment is merged into the parent.
-For the merge and deployment to be complete,
-you need to [allocate resources](/manage-resources/adjust-resources.md) to each of these newly added instances.
+By default, the resources from the child environment are also merged into the parent.
```

Comment:
It's actually not exactly like that.
The resource init strategy is only used for new apps/services/workers.

For instance if you have a php app + a mariadb service, and you would like to add redis.
You will create a new branch, add redis, and once ready, merge this branch to the parent to deploy the new service to the parent.
It's only the new redis service that will be initialized with the chosen strategy. The php app and the mariadb service won't be touched, because we don't want to change the resources of these already running service, whereas redis is a new service that is being created.

We are actually working on a change in the Console, to show the resources init option only if we detect that a new container will be deployed with the merge.
So the modal won't show the option and stay very simple (as it is today in production), and the option will only be displayed if it is relevant, because we detected that a new container will be created with the merge.

Let me know if you have questions 🙂

Reply:
@nicogommen Do you think this change in the Console might be ready for GA?